crates/swc/Cargo.toml

```diff
@@ -1,34 +1,37 @@
 
 
 [package]
-authors = ["강동윤 <kdy1997.dev@gmail.com>"]
-description = "Speedy web compiler"
+authors       = ["강동윤 <kdy1997.dev@gmail.com>"]
+description   = "Speedy web compiler"
 documentation = "https://rustdoc.swc.rs/swc/"
-edition = "2021"
-include = ["Cargo.toml", "src/**/*.rs"]
-license = "Apache-2.0"
-name = "swc"
-repository = "https://github.com/swc-project/swc.git"
-version = "0.212.3"
+edition       = "2021"
+include       = ["Cargo.toml", "src/**/*.rs"]
+license       = "Apache-2.0"
+name          = "swc"
+repository    = "https://github.com/swc-project/swc.git"
+version       = "0.212.3"
 
 [lib]
 bench = false
-name = "swc"
+name  = "swc"
 
 [features]
-concurrent = ["swc_ecma_transforms/concurrent", "swc_common/concurrent", "swc_ecma_minifier/concurrent", "swc_ecmascript/concurrent"]
-debug = ["swc_ecma_visit/debug"]
+# TODO: this seems unused, confirm & remove later once imports to swc flattens out ecma_*
+#concurrent = ["swc_ecma_transforms/concurrent", "swc_common/concurrent", "swc_ecma_minifier/concurrent", "swc_ecmascript/concurrent"]
```

Comment:
Can you remove `(swc, concurrent)` from ci?

Reply:
Oh right, thanks for catching. Updated.

Reply:
Peeking this more, it looks like this is still needed for now until we reorganize swc to detach some of its transitive deps. Reenabled this + with CI test again and then will try revisit this later.